src/components/definition-list/dt.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDt } from './definition-list.style';
+
+const Dt = ({
+  pr = 3,
+  label = 'Label',
```

Comment:
Should this have a default at all?

Reply:
@sjohnson-sage -  I had originally gave this as a default value so that if someone were to hash out a definition list, they'd see location of the Dt and the Dd. I'm not precious about it having a default value or not. Happy to remove if you think it is for the greater good?